third_party/iconv/BUILD.iconv.bazel

```diff
@@ -10,7 +10,7 @@ filegroup(
 configure_make(
     name = "iconv",
     configure_env_vars = select({
-        "@platforms//os:macos": {"AR": ""},
+        "@rules_rust//rust/platform:darwin": {"AR": ""},
```

Comment:
Is this change auto-generated, or manual?  If manual, why go with `rules_rust`'s version, rather than just `platforms`?

Reply:
Use of `@platform` here isn't supported in versions before... 3.6 if I recall correctly. Using the rust config settings guarantees the same level of compatibility the rules offers.

Reply:
Specifically because `@platforms` defines constraint_values and not config_settings.

Reply:
Looks like 3.7
>select() directly supports constraint_value with no need for an intermediate config_setting.

https://blog.bazel.build/2020/10/20/bazel-3-7.html

Reply:
Opened #373